.github/workflows/build.yml

```diff
@@ -7,6 +7,28 @@ on:
     branches: [ develop, master ]
 
 jobs:
+  build_arm64-binary: 
+     runs-on: ubuntu-latest
+     strategy:
+      matrix:
+        java: [ 8, 11 ]
+      fail-fast: false
+     steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        id: qemu
+        uses: docker/setup-qemu-action@v1
+      - name: Build arm-binary
+        run: |
+               docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+               arm64v8/ubuntu:20.04 \
+               bash -exc 'apt-get update -y && \
```

Comment:
`apt-get update'  if necessary?  Is it time-consuming?

Reply:
It is necessary here because without apt-get update it doesn't locate the maven package and falling with below..
![aaaaa](https://user-images.githubusercontent.com/40816837/193005208-61748200-cbc1-464f-a630-5afa6156b30f.jpg)

Reply:
make sense.